### ISSUE TITLE
Markdown in footnotes are not converted to html

### ISSUE THREAD

**Issue by Ziinc** [pragdave/earmark#373]
input markdown
```
here is my footnote[^1]

[^1]: here is some [information](http://google.com)
```
The main body gets converted, but the markdown text in the footnote does not.

**Comment by RobertDober** [comment]
Obviously you are the only one using footnotes :wink: , you detected quite an old regression :fearful: , thank you for the report

**Comment by RobertDober** [comment]
Will be fixed in https://github.com/RobertDober/earmark_parser/issues/1

**Comment by RobertDober** [comment]
fixed in EarmarkParser 1.4.10
